src/main/java/model/Admin.java

```diff
@@ -0,0 +1,44 @@
+package model;
+
+import model.exceptions.NonExistentTeachRequestException;
+import model.exceptions.TeacherNotSuitableForCourseException;
+
+import java.util.List;
+
+public class Admin extends Employee {
+    private List<Teacher> teacherList;
+    private List<Course> classes;
+    private TeachRequestMap teachRequestMap;
+
+    public Admin(String name, List<Course> classes, TeachRequestMap teachRequestMap) {
+        super(name);
+        this.classes = classes;
+        this.teachRequestMap = teachRequestMap;
+    }
+
+    public List<Teacher> findSuitableStaff(Course course) {
+        List<Teacher> suitableTeacherList = null;
```

Comment:
This needs to be a class(ArrayList<Teacher>,LinkedList<Teacher>), not an interface(List<Teacher>)